rs/tests/boundary_nodes/custom_domains/setup.rs

```diff
@@ -218,7 +218,7 @@ async fn setup_remote_docker_host(
 ) -> Result<DeployedUniversalVm, Error> {
     UniversalVm::new(REMOTE_DOCKER_HOST_VM_ID.into())
         .with_config_img(get_dependency_path(
-            "rs/tests/custom_domains_uvm_config_image.zst",
```

Comment:
Could you replace this with an environment variable and add something like the following to the `env` in the system-tests:

```
"CUSTOM_DOMAIN_UVM_CONFIG_PATH": "$(rootpath :custom_domains_uvm_config_image)",
```

I'm afraid that when I approve this @nmattia is going to force me to pay 3 rounds of beer for the team:

<img width="1057" alt="Screenshot 2024-10-18 at 16 10 04" src="https://github.com/user-attachments/assets/b1353b5c-bd77-4e7e-ac11-a187c726600e">

Now I can expense the beer but the 250 pushups are going to "problematic" ;)

Reply:
Okokok, you convinced me to change it :)